tutorials/understanding-feegrant/index.md

```diff
@@ -107,8 +107,8 @@ To avoid having to copy and paste the user addresses, now is a good time to expo
 :::
 
 ```sh
-$ export ALICE_KEY=$(simd keys show alice -a)
-$ export BOB_KEY=$(simd keys show bob -a)
+$ export ALICE_ADDRESS=$(simd keys show alice -a)
+$ export BOB_ADDRESS=$(simd keys show bob -a)
```

Comment:
```suggestion
$ export ALICE=$(simd keys show alice --address)
$ export BOB=$(simd keys show bob --address)
```

What do you think about just using `ALICE` and `BOB` if we use `--address` to make clear?